src/screens/Dag.js

```diff
@@ -8,6 +8,7 @@
 import React, { useCallback, useEffect, useRef, useState } from 'react';
 import txApi from '../api/txApi';
 import DagComponent from '../components/DagComponent';
+import { useNewUiEnabled } from '../hooks';
```

Comment:
These changes are also part of the other PR

Reply:
PR merged with master. Waiting Review.🙂